### DISCUSSION TITLE
Does this also work for unbound?

### DISCUSSION THREAD

**Discussion by FlamingBluejay** [mattwebbio/orbital-sync#265] [Q&A]
Hi,
I use an unofficial docker image of pi-hole which sets up unbound with pi-hole. Here is the image that I use:
image: cbcrowe/pihole-unbound:latest

Would there be a way to still use orbital-sync while preserving the ability to have unbound on both pi-hole servers?

**Comment by dsquare86** [comment]
Works fine for me. 
Primary is a Proxmox LXC
Secondary is a container on Mikrotik router - same image as you mentioned.
